pkg/workspace/controllers/workspace_controller.go

```diff
@@ -53,13 +55,27 @@ const (
 	revisionHashSuffix      = 5
 )
 
+var (
```

Comment:
create an individual package and move this definition and register code to that package.  "pkg/workspace/metrics/"